tests/test_transforms.py

```diff
@@ -424,3 +425,48 @@ def test_values_columns() -> None:
         sqlglot.parse_one("INSERT INTO cities VALUES ('Amsterdam', 1)").transform(values_columns).sql()
         == "INSERT INTO cities VALUES ('Amsterdam', 1)"
     )
+
+
+def test_sha256() -> None:
+    # snowflake default sha2 length is 256
+    assert (
+        sqlglot.parse_one("insert into table1 (name) select sha2('foo')").transform(sha256).sql(dialect="duckdb")
+        == "INSERT INTO table1 (name) SELECT SHA256('foo')"
+    )
+    assert (
+        sqlglot.parse_one("insert into table1 (name) select sha2_hex('foo')").transform(sha256).sql(dialect="duckdb")
+        == "INSERT INTO table1 (name) SELECT SHA256('foo')"
+    )
+
+    assert (
+        sqlglot.parse_one("insert into table1 (name) select sha2('foo', 256)").transform(sha256).sql(dialect="duckdb")
+        == "INSERT INTO table1 (name) SELECT SHA256('foo')"
+    )
+    assert (
+        sqlglot.parse_one("insert into table1 (name) select sha2_hex('foo', 256)")
+        .transform(sha256)
+        .sql(dialect="duckdb")
+        == "INSERT INTO table1 (name) SELECT SHA256('foo')"
+    )
+
+    # values with hash length other than 256 are not transformed
+    assert (
+        sqlglot.parse_one("insert into table1 (name) select sha2('foo', 224)").transform(values_columns).sql()
+        == "INSERT INTO table1 (name) SELECT SHA2('foo', 224)"
+    )
+    assert (
+        sqlglot.parse_one("insert into table1 (name) select sha2_hex('foo', 224)").transform(values_columns).sql()
+        == "INSERT INTO table1 (name) SELECT SHA2_HEX('foo', 224)"
+    )
+
+    # values with unrecognised args signature are not transformed
+    assert (
+        sqlglot.parse_one("insert into table1 (name) select sha2_hex()").transform(values_columns).sql()
+        == "INSERT INTO table1 (name) SELECT SHA2_HEX()"
+    )
+    assert (
+        sqlglot.parse_one("insert into table1 (name) select sha2_hex('foo', 256, 'wtf')")
+        .transform(values_columns)
+        .sql()
+        == "INSERT INTO table1 (name) SELECT SHA2_HEX('foo', 256, 'wtf')"
+    )
```

Comment:
would you mind also adding a test to test_fakes.py too? this will guarantee integrated test coverage that includes duckdb's execution